index.bs

```diff
@@ -192,6 +192,7 @@ interface ImageCapture {
     readonly attribute MediaSettingsRange         imageHeight;
     readonly attribute MediaSettingsRange         imageWidth;
     readonly attribute FrozenArray&lt;FillLightMode> fillLightMode;
+    readonly attribute FrozenArray&lt;DOMString>     supportedImageFormats;
```

Comment:
Could we add an entry to https://github.com/w3c/mediacapture-image/blob/master/implementation-status.md as well?